frontend/src/lib/components/displays/CheckpointDisplay.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+	import { GripVertical, Lock } from 'lucide-svelte';
+
+    function deleteCheckpoint() {
+        //todo
+    }
+
+</script>
+
+<div class="flex items-center p-1">
+	<GripVertical color="gray" class="h-10 w-10" />
+	<div
+		class="Learning-Unit-Display preset-filled-surface-100-900 rounded-border border-surface-200-800 flex w-full items-center rounded-lg border-[1px] p-3 text-base"
```

Comment:
Learning-Unit-Display class? what is up with these classes what is their purpose?